portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,59 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['I\'m ready, I\'m ready, I\'m ready...', 'You know what\'s funnier than 24? 25!', 'East?! I thought you said Weest!', 'Squidwaard, I used your clarinet to unclog my toilet!'];
+      ['I\'m ready, I\'m ready, I\'m ready...', 
+      'You know what\'s funnier than 24? 25!', 
+      'East?! I thought you said Weest!', 
+      'Squidwaard, I used your clarinet to unclog my toilet!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+}
+
+/**
+ * Reveal hidden description upon mouseover and hides the project title
```

Comment:
```suggestion
 * Reveals hidden description upon mouseover and hides the project title
```

When you're writing reference documentation for a method, phrase the main method description in terms of what the method does (gets, lists, creates, searches), rather than what the developer would use it to do (get, list, create, search).

It's a subtle distinction that manifests mostly in whether the initial verb in the description has an -s at the end or not.

See: https://developers.google.com/style/reference-verbs

Reply:
Also, please update your other comments to reflect this recommendation.

Reply:
Done!